docs/Migrating.md

```diff
@@ -1,17 +1,17 @@
-# Migrating from ML-Agents v0.3 to ML-Agents v0.4
+# Migrating from the ML-Agents toolkit v0.3 to the ML-Agents toolkit v0.4
```

Comment:
We can probably word this more concisely as "Migrating from ML-Agents toolkit v0.3 to v0.4`